### PR TITLE
Add code copied from trials

### PR DESCRIPTION
**Pull request by krisbobo**
In this section, I copied code from trials to use for code review.

### CODE REVIEW

**Review by krisbobo — COMMENTED** [review]
# Highlights #
- Good job so far in implementing the project requirements but some changes are still needed. 
- The code didn't pass the linter checks
- Descriptive PR 
 ## Required Changes ##
- I understand the need to complete the technical part of your work but documenting your work in a professional manner is equally as important. I suggest you document your work in a professional manner as it would help others and yourself understand your work later on down the line.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,441 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```
Comment (krisbobo) [inline index.html:4]: I suggest you add the `  <meta name="viewport" content="width=device-width, initial-scale=1.0">` to your head tag as it improves responsiveness.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+    <title>Document</title>
+    <link rel ="stylesheet" href="style.css">
+</head>
+<body><nav id="navbar">
+    <header>JS Documentation</header>
```
Comment (krisbobo) [inline index.html:9]: I suggest you place sections on new lines for easy readability.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+      <li><a class="nav-link" href="#Constants">Constants</a></li>
+      <li><a class="nav-link" href="#Data_types">Data types</a></li>
+      <li>
+        <a class="nav-link" href="#if...else_statement">if...else statement</a>
+      </li>
+      <li><a class="nav-link" href="#while_statement">while statement</a></li>
+      <li>
+        <a class="nav-link" href="#Function_declarations"
+          >Function declarations</a
+        >
```
Comment (krisbobo) [inline index.html:36]: I suggest you always try to make sure your opening and closing tags are properly closed in the same lines as it makes it easier for others to read your code and follow.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,441 @@
...
+        <code
+          >// THIS WILL CAUSE AN ERROR function f() {}; const f = 5; // THIS WILL
+          CAUSE AN ERROR ALSO function f() { const g = 5; var g; //statements
+          }</code
+        >
+        However, object attributes are not protected, so the following statement
+        is executed without problems.
+        <code
+          >const MY_OBJECT = {"key": "value"}; MY_OBJECT.key = "otherValue";</code
+        >
```
Comment (krisbobo) [inline index.html:264]: Good job using semantic HTML tags, I suggest you always make sure your tags are properly closed so as to ease readability.

**Review by m05e5 — APPROVED** [review]
# Hi @krisbobo , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**